config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -54,6 +54,7 @@ tolerations: []
 
 # Extra environment variables
 extraEnv: []
+envFrom: []
```

Comment:
Can you also add this to the helm chart README? https://github.com/aws/aws-node-termination-handler/blob/main/config/helm/aws-node-termination-handler/README.md?plain=1#L70